controller/src/beerocks/master/son_actions.cpp

```diff
@@ -505,3 +505,70 @@ bool son_actions::send_ap_config_renew_msg(ieee1905_1::CmduMessageTx &cmdu_tx, d
 
     return result;
 }
+
```

Comment:
I think the commit title doesn't stand for itself - would be really hard to differentiate this commit from other based on the title.
would expect the title to mention what is added (not just "method").
e.g.:
`controller: son_actions: add beacon_metrics_query method`